website/content/en/status/report-2023-01-2023-03/aarch64-kasan.adoc

```diff
@@ -8,14 +8,14 @@ They are particularly effective in combination with regression testing suites or
 Unlike tools such as Valgrind, software must be recompiled to enable a given sanitizer, but sanitizers can be used in the kernel.
 Kernels with sanitizers enabled incur a significant performance overhead from the runtime, in both CPU utilization and memory usage.
 
-As of gitref:89c52f9d59fa[repository=src], the kernel address sanitizer that was previously exclusive to amd64 has now been ported to arm64.
+As of gitref:89c52f9d59fa[repository=src], the kernel address sanitizer that was previously exclusive to amd64 is ported to arm64.
 
 Prior testing has been done on a decent variety of machines, including:
 
 - Various Ampere Altra machines
 - QEMU
 - Microsoft's "Volterra" Devkit
-- BHyve (WIP)
+- bhyve (WIP).
```

Comment:
I do not remember how we dealt with punctuation in lists last quarter. I prefer to leave status reports authors do what they prefer about it (I personally add a `;` at the end of each line element, except last one which gets a `.` if the sentence is finished).

If you want to add the `.`, I am not against, but please ensure at least that the listing style is consistent with whatever we did last quarter.

Reply:
Thanks. I sped through history for last quarter, found https://github.com/freebsd/freebsd-quarterly/commit/76535b3ef58c2ce63a8ee21b9267ccf035179602 with: 

>> * where a list is preceded by a sentence with a colon, close the list with a full stop.